### PR TITLE
SV-482 Add integration tests

### PR DESCRIPTION
**Pull request by ben1stone**
New pull request based off master

### CODE REVIEW

File: IntegrationTests/Courses API Tests.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+							"});\r",
+							"\r",
+							"var jsonData = pm.response.json();\r",
+							"\r",
+							"var total = jsonData.length;\r",
+							"var approvedStandards = jsonData.filter(obj => obj.status == 'Approved for delivery');\r",
+							"var retiredStandards = jsonData.filter(obj => obj.status == 'Retired');\r",
+							"var withdrawnStandards = jsonData.filter(obj => obj.status == 'Withdrawn');\r",
+							"var inDevelopmentStandards = jsonData.filter(obj => obj.status == 'In development');\r",
+							"var proposalStandards = jsonData.filter(obj => obj.status == 'Proposal in development')\r",
```
Comment (DevenKShah) [inline IntegrationTests/Courses API Tests.postman_collection.json:26]: missing semi colon at the end

File: IntegrationTests/Courses API Tests.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Successful GET standards from IFATE\", function () {\r",
+							"    pm.expect(pm.response.code).to.be.oneOf([200]);\r",
+							"});\r",
+							"\r",
+							"var jsonData = pm.response.json();\r",
+							"\r",
+							"var total = jsonData.length;\r",
```
Comment (DevenKShah) [inline IntegrationTests/Courses API Tests.postman_collection.json:21]: The status names are repeated in the other script, perhaps we can declare them in collection variables

File: IntegrationTests/CoursesApiTests.postman_environment.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+			"value": "https://localhost:5001/api/courses",
+			"enabled": true
+		},
+		{
+			"key": "adultCareWorkerIfateReference",
+			"value": "ST0005",
+			"enabled": true
+		},
+		{
+			"key": "adultCareWorkerStandardUId",
```
Comment (DevenKShah) [inline IntegrationTests/CoursesApiTests.postman_environment.json:21]: These can be declared in the collection variables as they are constants

**Review by DevenKShah — APPROVED** [review]
Looks good, couple of minor suggestions.

**Review by CodeWolf86 — APPROVED** [review]

**Pull request merged**